src/main/java/org/terasology/dynamicCities/parcels/DynParcelTypeHandler.java

```diff
@@ -15,41 +15,50 @@
  */
 package org.terasology.dynamicCities.parcels;
 
-
 import com.google.common.collect.ImmutableMap;
 import org.terasology.commonworld.Orientation;
 import org.terasology.math.geom.Rect2i;
-import org.terasology.persistence.typeHandling.DeserializationContext;
 import org.terasology.persistence.typeHandling.PersistedData;
 import org.terasology.persistence.typeHandling.PersistedDataMap;
 import org.terasology.persistence.typeHandling.RegisterTypeHandler;
-import org.terasology.persistence.typeHandling.SerializationContext;
-import org.terasology.persistence.typeHandling.SimpleTypeHandler;
+import org.terasology.persistence.typeHandling.*;
 
 import java.util.Map;
+import java.util.Optional;
 
 @RegisterTypeHandler
-public class DynParcelTypeHandler extends SimpleTypeHandler<DynParcel> {
+public class DynParcelTypeHandler extends TypeHandler<DynParcel>
+{
```

Comment:
According to style standards, open curly braces should be placed at the end of lines of code. So please run reformatting code tool for this Java file. If you are using IntelliJ `Ctrl + Alt + L` is enough.